src/controllers/BaseController.php

```diff
@@ -123,24 +123,25 @@ public function actionOrderCallback()
                 }
             }
 
-            echo 'File sync successful' . PHP_EOL;
+            $output .= 'File sync successful' . PHP_EOL;
         }
 
-        Craft::$app->end('OK');
+        Craft::$app->end("$output OK");
```

Comment:
@bhupeshappfoster this should be fixed. It should be `Craft::$app->end($output.'OK'); `